server/cabal.project.freeze

```diff
@@ -100,255 +101,266 @@ constraints: any.Cabal ==2.4.0.1,
              any.data-default-instances-containers ==0.0.1,
              any.data-default-instances-dlist ==0.0.1,
              any.data-default-instances-old-locale ==0.0.1,
-             any.data-dword ==0.3.1.2,
+             any.data-dword ==0.3.2,
              any.data-endian ==0.1.1,
              any.data-has ==0.3.0.0,
-             any.data-serializer ==0.3.4,
-             any.data-textual ==0.3.0.2,
+             any.data-serializer ==0.3.4.1,
+             any.data-textual ==0.3.0.3,
              any.deepseq ==1.4.4.0,
-             any.deferred-folds ==0.9.10,
+             any.deferred-folds ==0.9.10.1,
              any.dense-linear-algebra ==0.1.0.0,
              any.dependent-map ==0.2.4.0,
              any.dependent-sum ==0.4,
-             any.directory ==1.3.3.0,
-             any.distributive ==0.6,
+             any.directory ==1.3.6.0,
+             any.distributive ==0.6.2,
              distributive +semigroups +tagged,
-             any.dlist ==0.8.0.6,
+             any.dlist ==0.8.0.8,
              any.easy-file ==0.2.2,
              any.either ==5.0.1.1,
-             any.ekg-core ==0.1.1.6,
+             any.ekg-core ==0.1.1.7,
              any.ekg-json ==0.1.0.6,
-             any.entropy ==0.4.1.4,
+             any.entropy ==0.4.1.6,
              entropy -halvm,
              any.erf ==2.0.0.0,
              any.errors ==2.3.0,
-             any.exceptions ==0.10.2,
-             any.fast-logger ==2.4.15,
-             any.file-embed ==0.0.11,
+             any.exceptions ==0.10.4,
+             any.fail ==4.9.0.0,
+             any.fast-logger ==3.0.1,
+             any.file-embed ==0.0.11.2,
              any.filepath ==1.4.2.1,
              any.focus ==1.0.1.3,
-             any.foldl ==1.4.5,
-             any.free ==5.1.1,
+             any.foldl ==1.4.6,
+             any.free ==5.1.3,
              any.generic-arbitrary ==0.1.0,
-             any.ghc-boot-th ==8.6.5,
-             any.ghc-heap ==8.6.5,
-             any.ghc-heap-view ==0.6.0,
+             any.ghc-boot-th ==8.10.1,
+             any.ghc-heap ==8.10.1,
+             any.ghc-heap-view ==0.6.2,
              ghc-heap-view -prim-supports-any,
-             any.ghc-prim ==0.5.3,
+             any.ghc-prim ==0.6.1,
+             graphql-engine +developer,
              any.happy ==1.19.12,
              happy +small_base,
-             any.hashable ==1.2.7.0,
+             any.hashable ==1.3.0.0,
              hashable -examples +integer-gmp +sse2 -sse41,
-             any.hashtables ==1.2.3.1,
-             hashtables -bounds-checking -debug -portable -sse42 +unsafe-tricks,
-             any.haskell-lexer ==1.0.2,
-             any.hasql ==1.3.0.5,
-             any.hasql-pool ==0.5.0.2,
-             any.hasql-transaction ==0.7.1,
-             any.hedgehog ==0.6.1,
+             any.hashtables ==1.2.3.4,
+             hashtables -bounds-checking -debug -detailed-profiling -portable -sse42 +unsafe-tricks,
```

Comment:
Is it necessary to compile this with detailed profiling, even for non-dev applications?

Reply:
This file is just the output of `cabal freeze`; all these flags are the baked in defaults that the packaged ship with. Any fiddling we want to do with them in the future will live in `cabal.project` and have docs